internal/ocm/resource/cluster.go

```diff
@@ -108,6 +108,42 @@ func (c *Cluster) CreateNodes(autoScalingEnabled bool, replicas *int64, minRepli
 	return nil
 }
 
+func (c *Cluster) CreateHcpNodes(replicas *int64, computeMachineType *string,
+	labels map[string]string, availabilityZones []string, multiAZ bool, workerDiskSize *int64) error {
+	nodes := cmv1.NewClusterNodes()
```

Comment:
HCP has a dedicated NodePool Resource. Not super familiar with this TF module at the moment, but we should have a correspinding resource. We will have clusters with zero workers in the near future